modules/test/playwright/pages/portal-web/HomePage.ts

```diff
@@ -0,0 +1,26 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {Locator, Page} from '@playwright/test';
+export class HomePage {
+	readonly applicationsMenuButton: Locator;
+	readonly page: Page;
```

Comment:
The idea here is to isolate Locators within Pages, to keep concerns as separated as possible.

Reply:
Would it make sense to declare them private so that users of the page don't access them directly? No need to do it in this PR if you want to make progress, but I would like to hear your opinion...

Reply:
Good idea! :+1: And if somebody needs it for some odd reason, they can easily make it public. I'll make the change.

Reply:
Hmm after trying to implement it, I am no longer sure if this is a good idea. For each locator property that is private, we would need something like a public direct getter or action wrapper. It's simpler if we allow external access. Just isolating location in relevant page files seems to me like the most valuable part of this. Private can still be used when applicable, when the property is truly private.

Reply:
Yeah, if it needs a getter there's no point in making it private. My initial thought was that locators seem very low level and I wanted to avoid coupling tests (especially from other teams not owning the page) with page locators since it is almost like accessing the page directly (without POM).

But if the tests cannot be expressed without the locators, then there's no remedy... :shrug: